flink-formats/flink-orc/pom.xml

```diff
@@ -55,6 +55,14 @@ under the License.
 			<optional>true</optional>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-base</artifactId>
+			<version>${project.version}</version>
+			<scope>provided</scope>
+			<optional>true</optional>
```

Comment:
I guess the connector-base dependency is optional for all formats since it is probably only required when using a file-based implementation

Reply:
Are the formats actually using the flink-connector-base API? If not, then they shouldn't have a dependency on it, as this is covered by the connector.

Reply:
This particular format uses it directly:
https://github.com/apache/flink/blob/c6997c97c575d334679915c328792b8a3067cfb5/flink-formats/flink-orc/src/main/java/org/apache/flink/orc/AbstractOrcFileInputFormat.java#L23 
The situation seems to be identical to the file-connector-files dependency in this format, which is declared as provided and optional:
https://github.com/afedulov/flink/blob/40ca1773d5c19b3c12e802cabf25e510eab342c9/flink-formats/flink-orc/pom.xml#L68

Reply:
I wouldn't mark it as optional.